src/lib/libdylink.js

```diff
@@ -48,6 +48,17 @@ var LibraryDylink = {
     registerWasmPlugin();
     `,
   $preloadedWasm: {},
+
+  $replaceORIGIN__deps: ['$PATH'],
+  $replaceORIGIN: (parentLibPath, rpath) => {
```

Comment:
s/parentLibPath/parentLibName/ ? (path is sounds a little like it might be the directory in which that parent lives).